ietf-layer0-types.yang

```diff
@@ -1948,7 +2017,7 @@ module ietf-layer0-types {
         "OIF-400ZR-01.0: Implementation Agreement 400ZR";
     }
     leaf tx-polarization-power-difference {
-      type decimal-2-digits;
+      type decimal-2;
```

Comment:
use power-ratio instead ?

Reply:
it makes sense for me since power-ration is typedef of decimal-2 type

Reply:
Resoved in https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/82/commits/ad09ab3fbf5a49dec5e467d485c6099d31687db2